@xen-orchestra/xapi/network.mjs

```diff
@@ -0,0 +1,14 @@
+export default class Network {
+  async setMtu(network, mtu) {
+    await network.set_MTU(mtu)
+
+    // The MTU will not be updated for VIFs of paused VM, but we can't unplug/replug VIF on a paused VM
```

Comment:
what about suspended VM ? do they behave like Running or  Paused ?

Reply:
Suspended VMs' MTU is updated when they are resumed.